tuxemon/states/combat/combat.py

```diff
@@ -163,7 +163,10 @@ def startup(self, **kwargs):
         self.show_combat_dialog()
         self.transition_phase("begin")
         self.task(partial(setattr, self, "phase", "ready"), 3)
-
+        self.old_stats_data = []
```

Comment:
i don't think this should be here.  there is a concept of "a single source of truth", and this is spreading the "truth" around.  since techniques already have a copy of the old stats, then it doesn't need to be repeated here.  i would like to see this removed, since it is redundant.

Reply:
another issues is active_players.  it may change in a battle if there are multiple trainers, which is supported.  so that being a list of values with no reference to monsters/trainers makes it impossible to make it correct if trainers change.

Reply:
i believe this is redundant now?